plugins/elsys_sensor_parser/parser.js

```diff
@@ -98,7 +106,7 @@ let parser = new Parser()
             9: gpsParser,
             10: pulseCountParser,
             11: pulseCountAbsoluteParser,
-            12: temperatureParser,
```

Comment:
Should temperatureParser be removed?
As I read the assignment, both water is external and air is temperature?